packs/git/actions/clone.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: clone
+runner_type: run-remote
+description: "Clone a repository"
+enabled: true
+entry_point: ''
+parameters:
+  source:
```

Comment:
I guess this only works for oublic repos unless the system user has access to these repos - right? 

I am ok with this assumption but wanted to clarify if that is your intent and how it works.